ros/eloquent/ubuntu/bionic/ros1-bridge/Dockerfile

```diff
@@ -35,11 +35,6 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     ros-eloquent-demo-nodes-py=0.8.4-1* \
     && rm -rf /var/lib/apt/lists/*
 
-# install downstream packages
```

Comment:
Is this not necessary anymore ?

Reply:
https://github.com/osrf/docker_images/pull/478#discussion_r506654208